src/Commands/Env/DeployCommand.php

```diff
@@ -9,11 +9,20 @@ class DeployCommand extends TerminusCommand
 {
 
     /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
```

Comment:
@marktheunissen In PHP I'm afraid that null is not falsy. You have to check is_null($sites).